### ISSUE TITLE
Feature Request: Read WEBHOOK_URL from a secret instead of specificying in an env variable

### ISSUE THREAD

**Issue by farshad-hobsons** [aws/aws-node-termination-handler#170]
`WEBHOOK_URL` is treated as a secret/private by some organizations since it gives the ability to send a message to a group of people e.g. in a slack channel.

Currently the manifests in https://github.com/aws/aws-node-termination-handler/releases/download/v1.4.0/all-resources.yaml require to hard code the `WEBHOOK_URL` value which some organization can not afford to put in Github.

Please provide the flexibility in the published `all-resources.yaml` to use either of the options: e.g. either use `WEBHOOK_URL` or `WEBHOOK_URL_FROM_SECRET` where yaml can look like

```yaml
...
      - name: WEBHOOK_URL_FROM_SECRET
        valueFrom:
          secretKeyRef:
            name: secret-webhook-url
            key: address
```

**Comment by bwagner5** [comment]
Sounds reasonable to me. Not sure when we'll have the time to add this, but community contributions are welcome! I'll add a "Help-Wanted-Tag" to this issue if you or someone else wants to take a crack at it.

**Comment by imuqtadir** [comment]
I'll take a look and work on this.

**Comment by bwagner5** [comment]
this has been released in v1.6.0
